src/features/Apiexplorer/Dropdown/DropdownList/index.tsx

```diff
@@ -15,26 +21,25 @@ type TDropdownList = {
 
 const filterOptions = (options, query) => {
   query = query.toLowerCase();
-  return options.filter((option) => {
+  return Object.values(options).filter((option: TOption) => {
+    const lowerCaseTitle = option.title.toLowerCase();
```

Comment:
I think variable name should be `title` we dont need to write lowerCase with it.